GameData/Data.cs

```diff
@@ -27,5 +31,20 @@ public string GetGameIntroduction()
                 "When the gems are merged, it will the create an ancient weapon." +
                 "The Moonrake."; 
         }
+
+        public string MainCharacterNamePick()
```

Comment:
I think that all of these prompts should be part of the engine. This doesn't feel like it is specific to this game. What do you think?

Reply:
I think it might be part of the Game itself because we might use different name intros in different games,

> But nice idea, I will put those methods in the engine.